integration-tests/cli/state-import.it.js

```diff
@@ -0,0 +1,131 @@
+import { oneLine } from 'common-tags'
+import StateImport from '@commercetools/state-importer'
+import { getCredentials } from '@commercetools/get-credentials'
+import { clearData } from './helpers/utils'
+import states from '../../packages/state-importer/test/helpers/sampleStates.json'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'state-import-integration-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('State importer', () => {
+  let apiConfig
+  let stateImport
+
+  const logger = {
+    error: () => {},
+    warn: () => {},
+    info: () => {},
+    debug: () => {},
+  }
+
+  beforeAll(() =>
+    getCredentials(projectKey).then(credentials => {
+      apiConfig = {
+        host: 'https://auth.sphere.io',
+        apiUrl: 'https://api.sphere.io',
+        projectKey,
+        credentials,
+      }
+      // Clear all existing states
```

Comment:
oh sh*t... the above _looks dangerous_ and users/contributors of this tool can accidentally delete data from their project if configured incorrectly. Do we have a general __guide__ for developing/running this locally for example?

if not, can we add that?

Reply:
Hmmmm. This is valid. I will add a note to the `CONTRIBUTING.md` file that setup/teardown of integration tests could cause potential loss of data

Reply:
Added a note here: https://github.com/commercetools/nodejs/pull/504/commits/4cdbd8b1fcdcbcaecf0919077d4ce4c84d6f2b1f